### PR TITLE
"Frontend GQL updates to connect to backend"

### PR DESCRIPTION
**Pull request by richardcolachagua**

### CODE REVIEW

File: src/pages/MemberSettingsPage/ChangePassword/index.jsx
Comment (AlanisYates) [inline src/pages/MemberSettingsPage/ChangePassword/index.jsx:1]: We don't store passwords on the backend it's all done through firebase look into how to send a pw reset email with the firebase docs

File: src/pages/MemberSettingsPage/MemberSettingsForm/index.jsx [hunk 1]
```diff
@@ -36,21 +56,40 @@ const MemberSettingsForm = () => {
...
+          variables: values,
+        });
+        if (data.updateProfile) {
+          console.log("Profile updated successfully!");
+        } else {
+          console.log("Failed to update profile.")
+        }
+      } catch (error) {
+        console.error("An error occured:", error)
+      }
```
Comment (AlanisYates) [inline src/pages/MemberSettingsPage/MemberSettingsForm/index.jsx:88]: This looks Good

File: src/pages/MemberSettingsPage/MemberSettingsForm/index.jsx [hunk 2]
```diff
@@ -36,21 +56,40 @@ const MemberSettingsForm = () => {
...
 
   const formik = useFormik({
     initialValues: {
       firstName: "",
       lastName: "",
+      email: "",
       phone: "",
       address1: "",
       address2: "",
-      email: "",
```
Comment (AlanisYates) [inline src/pages/MemberSettingsPage/MemberSettingsForm/index.jsx:51]: Lets add in the initial value if they are already inputted that way they are shown to the user on first load

**Review by AlanisYates — CHANGES_REQUESTED** [review]

File: src/pages/MemberSettingsPage/ChangePassword/index.jsx
Comment (richardcolachagua) [reply src/pages/MemberSettingsPage/ChangePassword/index.jsx:1]: handled this going to submit a new pr

**Pull request closed without merging**